### PR TITLE
TCD/JDJ/RULE19-19

### PR DESCRIPTION
**Pull request by JacksonJ-KC**

### CODE REVIEW

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+            "ruleset_reference": "G3.1.2.9",
+            "rule_description": "For baseline systems 9 and 10 the system fan electrical power (Pfan) for supply, return, exhaust, and relief shall be  CFMs \u00d7 0.3, where, CFMs = the baseline system maximum design supply fan airflow rate, cfm. If modeling a non-mechanical cooling fan is required by Section G3.1.2.8.2, there is a fan power allowance of Pfan = CFMnmc \u00d7 0.054, where, CFMnmc = the baseline non-mechanical cooling fan airflow, cfm for the non-mechanical cooling. ",
+            "applicable_rmr": "Baseline RMR",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "primary_rule": "Yes",
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "baseline": {
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json:19]: Could you add the ```proposed``` case?

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json:120]: Could you add the ```proposed``` case?

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json:222]: Could you add the ```proposed``` case?

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json (hunk 1 above)
Comment (JacksonJ-KC) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json:72]: Added

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json (hunk 2 above)
Comment (JacksonJ-KC) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json:227]: Added

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json (hunk 3 above)
Comment (JacksonJ-KC) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_19.json:383]: Added

**Comment by yunjoonjung-PNNL** [comment]
Could we update this TCD to meet schema 0.0.29?

**Pull request merged**